### PR TITLE
update babel version, add error logs, and remove dist

### PR DESCRIPTION
**Pull request by jaswantcoder**
*Issue #, if available:*

https://github.com/amzn/node-jose_aws-kms-extension/security/dependabot/1

*Description of changes:*

1. Bumped babel version
2. Added some error logs
3. Removed dist

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/kms/kmsAsymmetricSigningKey.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export class KMSAsymmetricSigningKey {
       };
       return signingResult;
     } catch (error) {
+      console.error('Error while signing from KMS:', error);
```
Comment (mjsheikh) [inline src/kms/kmsAsymmetricSigningKey.ts:62]: [Nit] Space after semi colon

Comment (mjsheikh) [reply src/kms/kmsAsymmetricSigningKey.ts:62]: Applicable at all similar places

File: src/kms/kmsAsymmetricSigningKey.ts [hunk 2]
```diff
@@ -94,6 +95,7 @@ export class KMSAsymmetricSigningKey {
       };
       return verificationResult;
     } catch (error) {
+      console.error('Error while verifying from KMS:', error);
```
Comment (mjsheikh) [inline src/kms/kmsAsymmetricSigningKey.ts:98]: [Nit] This doesn't seem grammatically apt. Can be rephrased to "Error from KMS while verifying: " Or "Error thrown during verification by KMS: "

Comment (mjsheikh) [reply src/kms/kmsAsymmetricSigningKey.ts:98]: Similar pattern applicable at other error messages added.

**Review by mjsheikh — APPROVED** [review]

File: src/kms/kmsAsymmetricSigningKey.ts (hunk 2 above)
Comment (jaswantcoder) [reply src/kms/kmsAsymmetricSigningKey.ts:98]: thanks for your suggestion, however, going with current message for now which also works and PR to merge urgently.

**Pull request merged**